app/services/audio.js

```diff
@@ -501,19 +523,19 @@ export default Service.extend({
       this._trackPlayerEventForNpr({
         category: 'Engagement',
         action: 'Stream_Pause',
-        label: `Streaming_${get(audio, 'name')}`
+        label: `Streaming_${get(storyOrStream, 'name')}`
       });
     } else if (type === 'ondemand') {
       this._trackPlayerEventForNpr({
         category: 'Engagement',
         action: 'On_demand_audio_pause',
-        label: get(audio, 'audio')
+        label: get(storyOrStream, 'audio')
       });
     }
 
-    if (audio && get(audio, 'audioType') !== 'stream') {
+    if (storyOrStream) {
       // we're not set up to handle pause listen actions from streams atm
-      this.sendPauseListenAction(audio.id);
+      this.sendListenAction(storyOrStream, 'pause');
```

Comment:
Is there a reason we took out the check for a stream here? Does this comment need to be updated?